src/app/cube/browse/components/guideline-filter/guideline-filter.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="wrapper">
+    <h2>Choose Specific Guidelines or Standards</h2>
+    <hr />
+    <div class="grid">
+        <!-- Frameworks -->
+        <div>
+            <h3>Frameworks</h3>
+            <hr />
+            <div class="filter-wrapper" *ngIf="frameworkFilter">
+                <div class="filter filter-checkbox" id="filter-checkbox" *ngFor="let f of frameworkFilter.filters">
+                    <span class="checkbox-grid" [tip]="f.tip ? f.tip : ''" [tipDisabled]="!f.tip" tipPosition="right">
+                        <clark-checkbox attr.aria-label="{{ f.name }}" [value]="f.active" (checkboxChecked)="selectFrameworkOption(f.value)" (checkboxUnchecked)="deselectFrameworkOption(f.value)"></clark-checkbox>
+                        <span>{{ f.name }}</span>
+                    </span>
+                </div>
+            </div>
+        </div>
+        <!-- Guidelines -->
+        <div class="guideline-flex">
+            <div class="height-wrapper">
+                <div class="input-wrapper">
+                    <label for="search"><i class="fas fa-search"></i></label>
+                    <input id="search" placeholder="Guidelines or Standards" (keyup)="searchChange$.next($event.target.value)" />
+                </div>
+                <div *ngIf="guidelines && guidelines.length > 0" class="guideline-sort">
+                    <span>{{ total }} Compatible Guidelines</span>
+                    <span>
+                        <button (activate)="clearSelected()" class="button" *ngIf="selectedGuidelines && selectedGuidelines.length > 0">Clear Selected</button>
```

Comment:
Is this styled red when active?

Reply:
<img width="1081" alt="Screen Shot 2021-09-10 at 9 13 40 AM" src="https://user-images.githubusercontent.com/32078831/132858992-d83e25b1-28d6-4ed3-9afc-16c5ac4da087.png">